### PR TITLE
Plugin Details: adds sidebar usps.

### LINKED ISSUE #57758: Plugin Details: Sidebar items
We can add the Sidebar items as static texts for now since we will only have WOO extensions

SS 2021-11-08 at 17 17 02

### PR DESCRIPTION
**Pull request by cpapazoglou**
#### Changes proposed in this Pull Request

* added static sidebar USPs for paid plugins.
* removed unused svg
* added "accordion" as search keyword for `FoldableCard` component.
* demo & documentation will not yet appear as they don't exist in the response which will be added here https://github.com/Automattic/wp-calypso/issues/58701. But they have been implemented 

#### Testing instructions

|Desktop | Mobile|
|-------|------|
|SS 2021-12-22 at 12 23 52|SS 2021-12-22 at 12 24 35|

* visit a paid plugin
* check that USPs appear (demo & documentation will not yet appear as they don't exist in the response) 
* visit a free plugin
* check that plugin details appear

Fixes #57758

### CODE REVIEW

File: client/my-sites/plugins/plugin-details-sidebar/index.jsx [hunk 1]
```diff
@@ -15,7 +14,39 @@ const PluginDetailsSidebar = ( props ) => {
...
+				) }
+				{ /* Needs to check for documentation_url */ },
+				{ isMarketplaceProduct && (
+					<PluginDetailsSidebarUSP
+						title={ translate( 'Support' ) }
+						description={ translate( 'Handled by WooCommerce.' ) }
+						last
+						links={ [
+							{ href: 'docs', label: translate( 'View documentation' ) },
+							{ href: 'https://automattic.com/privacy/', label: translate( 'See privacy policy' ) },
```
Comment (a8ci18n) [inline client/my-sites/plugins/plugin-details-sidebar/index.jsx]: :information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **102** times:
`translate( 'privacy-policy' )` *ES Score: 6*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/plugin_details_sidebar_items)

File: client/my-sites/plugins/plugin-details-sidebar/index.jsx [hunk 2]
```diff
@@ -1,40 +1,96 @@
...
 						</div>
-						<div className="plugin-details-sidebar__tested-value value">{ tested }</div>
-					</div>
+					) }
+				</div>
+			</>
+		);
+	}
+	const supportLinks = [
+		{ href: 'https://automattic.com/privacy/', label: translate( 'See privacy policy' ) },
```
Comment (a8ci18n) [inline client/my-sites/plugins/plugin-details-sidebar/index.jsx:50]: :information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **102** times:
`translate( 'privacy-policy' )` *ES Score: 6*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/plugin_details_sidebar_items)

**Review by gavande1 — APPROVED** [review]
LGTM 👍

**Comment by a8ci18n** [comment]
This Pull Request is now available for translation here: https://translate.wordpress.com/deliverables/7056042

Thank you @cpapazoglou for including a screenshot in the description! This is really helpful for our translators.

**Comment by a8ci18n** [comment]
Translation for this Pull Request [has now been finished](https://translate.wordpress.com/deliverables/overview/7056042).

**Pull request merged**